Sources/SectionScrubber.swift

```diff
@@ -76,6 +98,10 @@ public class SectionScrubber: UIView {
             if let scrubberImage = self.scrubberImage {
                 self.scrubberWidth = scrubberImage.size.width
                 self.scrubberImageView.image = scrubberImage
+                // scrubber is hidden by default
```

Comment:
Ha. I don't see why we should have it, maybe I'm missing something out. If there's a reason for this comment, maybe we should have the reason instead of this obvious fact.